Makefile

```diff
@@ -62,3 +62,9 @@ dist: clean
 
 release: dist
 	twine upload -r lukkien dist/*
+
+sandbox_requirements:
```

Comment:
we can modify the `make requirements`. I don't see it's being used. So essentially, we can rename `make develop` to `make sandbox` or even create an alias (If we prefer to keem the name to `sandbox`).